docs/config.rst

```diff
@@ -1,5 +1,6 @@
-Config Options
-**************
+================
+ Config Options
+================
```

Comment:
What is the point of these heading changes? If there is a useful point, then open a separate PR for documentation improvements.

Reply:
Reverted